mu-plugins/blocks/query-filter/postcss/style.pcss

```diff
@@ -0,0 +1,314 @@
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal !important;
+
+	& + label {
+		display: block;
+		margin: 0;
+		padding-block: var(--wp--custom--wporg-query-filters--spacing--padding--block);
+		padding-left: var(--wp--custom--wporg-query-filters--spacing--padding--inline-start);
+		padding-right: calc(var(--wp--custom--wporg-query-filters--spacing--padding--inline-end) * 2 + var(--wporg-query-filter--icon-size));
+	}
+
+	&:focus + label {
+		outline: 1.5px solid var(--wp--custom--wporg-query-filters--toggle--focus--color--outline);
+		outline-offset: -1.5px;
+		box-shadow: inset 0 0 0 3px var(--wp--custom--wporg-query-filters--toggle--focus--color--inset);
+	}
+
+	&:checked + label {
+		border-radius: 2px;
+		background-color: var(--wp--custom--wporg-query-filters--option--active--color--background);
+		color: var(--wp--custom--wporg-query-filters--option--active--color--text);
+		/* stylelint-disable-next-line function-url-quotes */
+		background-image: url("data:image/svg+xml,%3Csvg width='24' height='24' viewBox='0 0 24 24' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M12 13.0607L15.7123 16.773L16.773 15.7123L13.0607 12L16.773 8.28772L15.7123 7.22706L12 10.9394L8.28771 7.22705L7.22705 8.28771L10.9394 12L7.22706 15.7123L8.28772 16.773L12 13.0607Z' fill='%231E1E1E'/%3E%3C/svg%3E%0A");
+		background-repeat: no-repeat;
+		background-position: right var(--wp--custom--wporg-query-filters--spacing--padding--inline-end) center;
+	}
+}
+
+.wporg-query-filter__modal-actions {
+	display: flex;
+	justify-content: space-between;
+	padding: var(--wp--preset--spacing--10);
+	border-top:
+		var(--wp--custom--wporg-query-filters--border--width)
+		var(--wp--custom--wporg-query-filters--border--style)
+		var(--wp--custom--wporg-query-filters--border--color);
+}
+
+.wporg-query-filter__modal-actions input {
+	padding: 12px;
+	font-weight: var(--wp--custom--button--typography--font-weight);
+	font-size: var(--wp--custom--button--typography--font-size);
+	line-height: var(--wp--custom--button--typography--line-height);
+	color: var(--wp--custom--button--color--text);
+	background-color: var(--wp--custom--button--color--background);
+	border: none;
+	border-radius: 2px;
+
+	&:hover,
+	&:focus {
+		color: var(--wp--custom--button--hover--color--text);
+		background-color: var(--wp--custom--button--hover--color--background);
+	}
+
+	&:focus {
+		background-color: var(--wp--custom--button--color--background);
+		box-shadow: inset 0 0 0 3px var(--wp--preset--color--white);
+		outline: 1.5px solid var(--wp--custom--button--focus--border--color);
+		outline-offset: -1.5px;
+	}
+
+	&:active {
+		color: var(--wp--custom--button--active--color--text);
+		background-color: var(--wp--custom--button--active--color--background);
+	}
+
+	&.wporg-query-filter__modal-action-clear {
+		background-color: transparent;
+		color: #a13f2a; /* pomegrade-1 fails contrast against white. */
```

Comment:
@WordPress/meta-design I meant to flag this in the description— the mockup uses "Pomegrade" on white but that [fails to meet contrast requirements](https://webaim.org/resources/contrastchecker/?fcolor=E26F56&bcolor=FFFFFF). There's a note in the design library about darker versions of colors, but I didn't see any, so I've made this one up. If you'd rather something different, it just needs to have [a 4.5+ value against light-grey-2](https://webaim.org/resources/contrastchecker/?fcolor=A13F2A&bcolor=F6F6F6), since I'm using that as the hover color.

<img width="358" alt="Screenshot 2023-09-06 at 11 02 36 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/541093/389c6048-8246-4144-8538-dcfd74c04af1">

Hover:
<img width="376" alt="Screenshot 2023-09-06 at 11 02 50 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/541093/cf67c069-0d4d-4bf1-8517-48c26254c0c6">